core/convenience.js

```diff
@@ -64,8 +64,8 @@
     rp.key = password;
 
     /* do the encryption */
-    if (p.mode === "ccm" && sjcl.arrayBuffer && sjcl.arrayBuffer.ccm && plaintext instanceof ArrayBuffer) {
-      p.ct = sjcl.arrayBuffer.ccm.encrypt(prp, plaintext, p.iv, adata, p.ts);
+    if (p.mode === "ccm" && sjcl.arrayBuffer && sjcl.arrayBuffer.ccm) {
+      p.ct = sjcl.arrayBuffer.ccm.compat_encrypt(prp, plaintext, p.iv, adata, p.ts);
```

Comment:
Which operations do you mean? Why not use it in the convenience function? If not, library user would have to take care of the serialization of the ciphertext and all other data (iv, ts...) by himself.